internal/node/node_repositories.bzl

```diff
@@ -41,13 +42,15 @@ def node_repositories(**kwargs):
         minimum_bazel_version = "4.0.0",
     )
 
-    # buildifier: disable=print
-    print("""WARN: node_repositories is deprecated, please instead use:
-        load("@rules_nodejs//nodejs:repositories.bzl", "nodejs_register_toolchains")
-        nodejs_register_toolchains(name = "nodejs")
-
-        See https://github.com/bazelbuild/rules_nodejs/wiki/Migrating-to-5.0
-    """)
+    # Back-compat: allow yarn_repositories args to be provided to node_repositories
+    yarn_args = {}
```

Comment:
based on working with the repo recently, I think it's too soon to deprecate the build_bazel_rules_nodejs repositories helper. Too spammy to log here, and too much work for users, with no benefit in the vast majority case where they don't care about toolchains